packages/amplication-client/src/Resource/ResourceForm.tsx

```diff
@@ -67,9 +73,15 @@ function ResourceForm({ match }: Props) {
           },
           resourceId: resourceId,
         },
-      }).catch(console.error);
+      })
+        .then(() => {
+          if (currentResource?.resourceType === models.EnumResourceType.ProjectConfiguration) {
+            refetchProjects()
+          }
+        })
+        .catch(console.error);
```

Comment:
There is a built-in support in apollo client to reftech query - 
please have a look at this
https://github.com/amplication/amplication/blob/198ba57d95a08fb61118cebff509ac7cff2c6477/packages/amplication-client/src/VersionControl/Commit.tsx#L52